baldur/src/main/webapp/WEB-INF/web.xml

```diff
@@ -23,20 +23,23 @@
             <param-value>
                 org.opencadc.inventory.permissions
                 org.opencadc.baldur
+                ca.nrc.cadc.util
+                ca.nrc.cadc.rest
+                ca.nrc.cadc.vosi
             </param-value>
         </init-param>
```

Comment:
Having those permissions in web.xml doesn't work well with open source code.  I'll add the value for CADC operations for now.

Reply:
It would be better to use the alternative config file instead of hard coded group here. That allows for users and/or groups to be specified and users can change log level when A&A system (ac and cred for CADC) is not working.
If you want to leave it for now and create an issue to change it (and the other services) later that's fine.